### ISSUE TITLE
Updating the diffusers seems to make the model more aggressive 

### ISSUE THREAD

**Issue by nanafy** [d8ahazard/sd_dreambooth_extension#80]
Hi. Thank you for putting this repo together. I just wanted to bring to your attention that updating the diffusers==0.6.0 to 0.8.0. dev0 seems to make my trained models much more aggressive, to the point of no flexibility. My inputs are very similar to my outputs and the prompt instance needs to move to the back of the prompt structure. Mind you, I have been training my models without classification images. But I just wanted to let you know that I reverted to 62ff2f377ffc2784c22a38e24ba2236dbc8a5cfc as that is where I get the best trained results.

**Comment by d8ahazard** [comment]
Hey, thanks for the feedback. Unfortunately, this is probably not something I'm likely to change, as the reason for using Shivam's diffusers in the first place was to address several other issues with the "regular" diffusers.

A few suggestions:

Definitely mess with classifiers. It helps.

Use less training steps, or save a checkpoint more frequently, then go back with the same seed and see which one gives the best results. IMHO, aggressive means it's a good training method if it can get good results quickly. Also, adjusting the learning rate might help. 

This would be a great topic for the (discussions)[https://github.com/d8ahazard/sd_dreambooth_extension/discussions] page. ;)

**Comment by nanafy** [comment]
Thank you for all your hard work. I wish I knew more of this type of code. I mainly just do mobile UI stuff. I just wanted to bring it to your attention. I fully understand moving the project forward to the latest versions. I will try adding classifier images, I was just being lazy and found 0.6 to work pretty well without anything else. Going to keep a copy that commit locally and play around as the new version progresses to see how it best responds.
